x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/agg/PercentilesAgg.java

```diff
@@ -6,25 +6,31 @@
 package org.elasticsearch.xpack.sql.querydsl.agg;
 
 import org.elasticsearch.search.aggregations.AggregationBuilder;
+import org.elasticsearch.search.aggregations.AggregationBuilders;
 import org.elasticsearch.search.aggregations.metrics.PercentilesAggregationBuilder;
+import org.elasticsearch.search.aggregations.support.ValuesSourceAggregationBuilder;
 
 import java.util.List;
+import java.util.function.Function;
 
-import static org.elasticsearch.search.aggregations.AggregationBuilders.percentiles;
-
-public class PercentilesAgg extends LeafAgg {
+public class PercentilesAgg extends DefaultAggSourceLeafAgg {
 
     private final List<Double> percents;
 
     public PercentilesAgg(String id, AggSource source, List<Double> percents) {
         super(id, source);
         this.percents = percents;
     }
-
+    
```

Comment:
```suggestion

```

Reply:
I'm not sure why that was added since on my end there are no extra characters appearing ; maybe it's a CR/LF handling even though I have the autoCrLF setting enabled in git...
I've removed the line altogether.